app/pages/sections/catalogue-solutions/order-item/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "info for",
+  "description": "same description for every page",
```

Comment:
change this to
Provide the delivery date and price you've agreed with the supplier and the quantity you want to order.